CONTRIBUTING.md

```diff
@@ -30,24 +41,37 @@ mamba activate xdem-dev  # Or any other name specified above
 ### Tests
 
 At least one test per feature (in the associated `tests/test_*.py` file) should be included in the PR, using `pytest` (see existing tests for examples).
+The structure of test modules and functions in `tests/` largely mirrors that of the package modules and functions in `xdem/`.
 
 To run the entire test suite, run `pytest` in the current directory:
 ```bash
 pytest
 ```
 
+Running `pytest` will trigger a script that automatically downloads test data from [https://github.com/GlacioHack/xdem-data](https://github.com/GlacioHack/xdem-data) before running all tests.
+
+### Documentation
+
+If your changes need to be reflected in the documentation, update the related pages located in `doc/source/`. The documentation is written in MyST markdown syntax, similar to GitHub's default Markdown (see [MyST-NB](https://myst-nb.readthedocs.io/en/latest/authoring/text-notebooks.html) for details).
+
+To ensure that the documentation is building properly after your changes, you can run `pytest tests/test_doc.py` on Linux. On Windows and Mac, the documentation is not maintained, so you can wait to open the PR for it to be checked by the CI.
+
 ### Formatting and linting
 
-Install and run `pre-commit` (see [pre-commit documentation](https://pre-commit.com/)), which will use `.pre-commit-config.yaml` to verify spelling errors,
-import sorting, type checking, formatting and linting.
+Install and run `pre-commit` (such as with `mamba install pre-commit`, see [pre-commit documentation](https://pre-commit.com/) for details),
+which will use `.pre-commit-config.yaml` to verify spelling errors, import sorting, type checking, formatting and linting:
 
-You can then run pre-commit manually:
 ```bash
-pre-commit run --all-files
+pre-commit run --all
 ```
 
+You can then commit and push those changes.
 Optionally, `pre-commit` can be installed as a git hook to ensure checks have to pass before committing.
 
+### Final steps
```

Comment:
Maybe add a word that the tests have to succeed and the PR be approved for it to be merged.

Reply:
Done!